input/examples/condition-masked.xml

```diff
@@ -1,5 +1,5 @@
 <Condition xmlns="http://hl7.org/fhir">
-  <id value="cellulitis"/>
+  <id value="condition-masked"/>
```

Comment:
This is a poor example of masked. Only one element is masked. Redo.

Reply:
change not in PR yet.

Reply:
Please check again

Reply:
1. changes to condition-masked-subject: do not agree with adding a masked subject into AU Core example set. Suggest choose an Observation. If condition is kept, mask the category and code, and potentially other elements but not subject.
2. Addition of 2nd masked example not addressed - choose Observation, mask multiple fields including date.

Reply:
1. Addressed
2. Addressed